01_Libraries/ghost_v5_interfaces/include/ghost_v5_interfaces/util/device_config_factory_utils.hpp

```diff
@@ -42,6 +42,11 @@ namespace util
  *
  * port_configuration:
  *              use_partner_joystick = false/true
+ *      adi:
+ *          limit_switch_1:
+ *              port: A
+ *              type: DIGITAL
```

Comment:
type and io_type seems redundant. Can we consolidate into single DIGITAL_IN / DIGITAL_OUT

Reply:
This got resolved.